src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -310,18 +321,22 @@
 		Skript.registerEvent("Portal Create", SimpleEvent.class, PortalCreateEvent.class, "portal creat(e|ion)")
 				.description("Called when a portal is created, either by a player or mob lighting an obsidian frame on fire, or by a nether portal creating its teleportation target in the nether/overworld.",
 						"Please note that it's not possible to use <a href='../expressions.html#ExprEntity'>the player</a> in this event.")
-				.examples("")
+				.examples("on portal create:")
 				.since("1.0");
 		Skript.registerEvent("Projectile Hit", SimpleEvent.class, ProjectileHitEvent.class, "projectile hit")
 				.description("Called when a projectile hits an entity or a block.",
 						"Use the <a href='#damage'>damage event</a> with a <a href='../conditions.html#CondIsSet'>check</a> for a <a href='../expressions/#ExprEntity'>projectile</a> " +
 								"to be able to use the <a href='../expressions.html#ExprAttacked'>entity that got hit</a> in the case when the projectile hit a living entity.",
 						"A damage event will even be fired if the damage is 0, e.g. when throwing snowballs at non-nether mobs.")
-				.examples("")
+				.examples("on projectile hit:",
+						"\tevent-projectile is snowball",
+						"\tkill event-projectile")
```

Comment:
I will say use `delete` instead of `kill`, but already snowball would disappear. You should use arrow here.